recipes/powerplantmatching/meta.yaml

```diff
@@ -0,0 +1,60 @@
+{% set name = "powerplantmatching" %}
+{% set version = "0.4.0" %}
+{% set sha256 = "f84b8b715e00a6bd74c340a04f71bbd00c6e959916e4b9d85e6450358cf2f84e" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vvv "
+  noarch: python
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - numpy
+    - scipy
+    - pandas>=0.23.0
+    - networkx>=1.10
+    - pycountry
+    - xlrd
+    - seaborn
+    - pyyaml
+    - requests
+    - matplotlib
+    - geopy
+    - pip:
+      - entsoe-py
+
+test:
+  imports:
+    - powerplantmatching
+
+about:
+  home: https://github.com/FRESNA/powerplantmatching
+  license: GPL-3.0
+  license_family: GPL
+  license_file: LICENSE.txt
+  summary: Toolset for generating and managing Power Plant Data
+  description: |
+    This package provides ready-to-use power plant data for the European 
+    power system. Starting from openly available power plant datasets, 
+    the package cleans, standardizes and merges the input data to create 
+    a new combining dataset, which includes all the important information. 
+    The package allows to easily update the combined data as 
+    soon as new input datasets are released.
+  doc_url: https://github.com/FRESNA/powerplantmatching
+  dev_url: https://github.com/FRESNA/powerplantmatching
+
+extra:
+  recipe-maintainers:
+    - FabianHofmann
+    - coroa
```

Comment:
@coroa, are you OK with being a maintainer?

Reply:
Yes, thanks!